plugins/postcss-design-tokens/README.md

```diff
@@ -0,0 +1,259 @@
+# PostCSS Design Tokens [<img src="https://postcss.github.io/postcss/logo.svg" alt="PostCSS Logo" width="90" height="90" align="right">][postcss]
+
+[<img alt="npm version" src="https://img.shields.io/npm/v/@csstools/postcss-design-tokens.svg" height="20">][npm-url] [<img alt="Build Status" src="https://github.com/csstools/postcss-plugins/workflows/test/badge.svg" height="20">][cli-url] [<img alt="Discord" src="https://shields.io/badge/Discord-5865F2?logo=discord&logoColor=white">][discord]
+
+[PostCSS Design Tokens] lets you use design tokens in your CSS source files.
+
+```json
+{
+	"color": {
+		"background": {
+			"primary": { "value": "#fff" }
+		}
+	},
+	"size": {
+		"spacing": {
+			"small": { "value": "16px" }
+		}
+	}
+}
+```
+
+```pcss
+@design-tokens url('./tokens.json') format('style-dictionary3');
```

Comment:
I wonder if we should default to `style-dictionary3` to avoid the need to add this for convenience? Specially given there's no extra systems right now

Reply:
Perhaps this could be a plugin option with that as default?

Reply:
`design-token` is still too long imo. Can't we go for something shorter like `token()`?

Reply:
It is easy to mistake default for "best practice" and I'd rather not promote a single format as they are all problematic at the moment :/

If there ever is a format that is truly good for code authors I want to make that one the default, but `style-dictionary3` wont be it.

Making this a plugin option will make code less portable.
If you copy/paste between two projects you will need to make sure they both have the same default configured.

Reply:
That makes sense,  given that most usages will rely on `postcss-import` it's not something you're going to write often.